src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -985,7 +1010,7 @@ trait BronzeTransforms extends SparkSessionWrapper {
     // Build root level eventLog path prefix from clusterID and log conf
     // /some/log/prefix/cluster_id/eventlog
     val allEventLogPrefixes =
-    if(isMultiWorkSpaceDeployment) {
+    if(isMultiWorkSpaceDeployment && organisationId != spark.conf.get("spark.databricks.clusterUsageTags.clusterOwnerOrgId")) {
```

Comment:
instead of `spark.conf.get("spark.databricks.clusterUsageTags.clusterOwnerOrgId")` can we just use the standard getOrgId func so as not to duplicate this critical snippet?

Reply:
getOrgId function uses dbutils.notebook.getContext.tags("orgId") which sometime gives null pointer exception. I think spark.conf.get("spark.databricks.clusterUsageTags.clusterOwnerOrgId") is more robust then dbutils.notebook.getContext.tags("orgId")

Reply:
Initializer.getOrgId -- this should never give null pointer, right?

Reply:
PR #708 updated the logic

Reply:
implemented